src/diffusers/schedulers/scheduling_sde_ve.py

```diff
@@ -98,6 +100,9 @@ def get_adjacent_sigma(self, timesteps, t):
         raise ValueError(f"`self.tensor_format`: {self.tensor_format} is not valid.")
 
     def set_seed(self, seed):
```

Comment:
We should not work with `set_seed(...)` here as it goes against the `generator` design. IMO passing generators around is the correct thing to do here because:
- We cannot really pass seeds around and globally set `manual_seed(...)` every time: 
   - It's not a great design to retrieve the current seed from a generator and then pass this -> better to pass generator directly
   - Flax passes PNRG keys around that can be split -> we cannot split PyTorch seeds in the same way and we cannot pass a `seed` into a `torch.randn(...)` function -> so let's go for the generator here